Core/include/Acts/Seeding/SeedFinderConfig.hpp

```diff
@@ -119,6 +119,10 @@ struct SeedFinderConfig {
   // which will make seeding very slow!
   float rMin = 33 * Acts::UnitConstants::mm;
 
+  // z of last layers to avoid iterations
+  float zMinLastLayer = -2800 * Acts::UnitConstants::mm;
```

Comment:
as a general comment from my side: I find the seeding configuration to already have a worryingly high number of parameters to set (some of which look redundant or are not clear what they do). This PR will add two more to them.
Should we found a way to compress these configurations?

What I mean is: use a `std::pair<float, float>` instead of two `float` for defining a range for instance